pkg/podimpersonation/podimpersonation.go

```diff
@@ -559,9 +568,34 @@ func (s *PodImpersonation) augmentPod(pod *v1.Pod, sa *v1.ServiceAccount, secret
 				continue
 			}
 
+			vmount := v1.VolumeMount{
+				Name:      "user-kubeconfig",
+				MountPath: "/tmp/.kube",
+			}
+			cfgVMount := v1.VolumeMount{
+				Name:      "user-kube-configmap",
+				MountPath: "/home/.kube/config",
+				SubPath:   "config",
+			}
+
+			pod.Spec.InitContainers = []v1.Container{
+				{
+					Name:            "init-kubeconfig-volume",
+					Image:           s.imageName(),
```

Comment:
This should probably respect the[ image override that is done further down](https://github.com/rancher/steve/blob/dcea050d923061f427bb3f7e0a6a8f2a2d2992b9/pkg/podimpersonation/podimpersonation.go#L606-L609) in this function.

Reply:
Done